scripts/bert/staticbert/static_export_base.py

```diff
@@ -0,0 +1,198 @@
+"""
+Export Base Static Model (BERT)
+
+=========================================================================================
+
+This will export the base BERT model to a static model suitable for use in MXNet Module API.
```

Comment:
Thanks for your contribution! Would you mind adding some documentation to `index.rst`, and a test case in `https://github.com/dmlc/gluon-nlp/blob/master/scripts/tests/test_scripts.py`?

Reply:
Sure - no problem. I'll do that shortly.